src/g_game.c

```diff
@@ -1290,7 +1290,7 @@ boolean G_Responder(event_t* ev)
       // of demo playback, or if automap active.
       // Don't suck up keys, which may be cheats
 
-      return gamestate == GS_DEMOSCREEN &&
+      return (gamestate == GS_DEMOSCREEN || demoplayback) &&
```

Comment:
So this means that when playing a demo, we can't type idrate or idmypos anymore, unless the game is paused or the automap is active. Would something here be better?

https://github.com/fabiangreffrath/woof/blob/a21559ea95baec3f41749d603195adf1dcc9f8ea/src/mn_menu.c#L2739

Reply:
Maybe something like this?
```c
    // Pop-up Main menu?

    if (!menuactive)
    {
        if ((demoplayback && (action == MENU_ENTER || action == MENU_BACKSPACE))
            || action == MENU_ESCAPE) // phares
        {
```

Reply:
Done here:  https://github.com/fabiangreffrath/woof/pull/1582/commits/6499439693a06100f372020f462df110c7d1092d

Although `IDRATE` still works, even with the menu enabled.

Reply:
Looks good but it was a quick suggestion so I'm not sure if there are any "gotchas".